sdk/cosmos/azure-cosmos/azure/cosmos/aio/_database.py

```diff
@@ -132,7 +133,7 @@ async def read(
         *,
         initial_headers: Optional[Dict[str, str]] = None,
         **kwargs: Any
-    ) -> Dict[str, Any]:
+    ) -> CosmosDict:
```

Comment:
Does this mean we were already returning the `CosmosDict` here, but we just never updated the type hints?

Reply:
yep seems to be the case - probably because the initial CosmosDict/ CosmosList was only targeting item-level operations